src/components/generics/links/AddLink.vue

```diff
@@ -2,9 +2,13 @@
   <!-- We alawys display this link, even when user is not logged
     In this case, editions vues will redirect user to /auth -->
   <router-link :to="{name:documentType + '-add', params:{lang:$language.current}, query:query}">
-    <slot>
+    <template v-if="isShortLinkOnMobile">
```

Comment:
Arf, I was too fast on this one.

The logic of this component is to let devs the choice : 

* use default content : text creationTitle
* or explicitly set the content

It may not depend on mobile/not mobile.

See this for instance : 

![image](https://user-images.githubusercontent.com/11915659/55009979-b8510a00-4fe3-11e9-988d-3054ae46f3cb.png)

I will do a Pr with a fix, but I will pay attention to not duplictae the link :)